### DISCUSSION TITLE
Got an error "Your billing plan does not support private pages."

### DISCUSSION THREAD

**Discussion by LaFocus** [community/community#77528] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I've tried to configure github pages and got this error:
![image](https://github.com/community/community/assets/116096845/d29176b3-8a73-45c1-9bdb-2753bd6ff138)
I haven't change the repository setting to private since i've created it. How can i solve this problem?
I've tried to change settings to private and then to public back, but it didn't work

**Comment by queenofcorgis** [comment]
Hi there - would you mind clarifying if you had initially create a private page and what type of plan you are on?

**Comment by salvaRC** [comment]
Hello, I'm having the same problem. I'm on pro plan.

**Reply by queenofcorgis** [reply]
@salvaRC are you still experiencing this?

**Reply by yoannchaudet** [reply]
I suspect this is related to an issue we had regarding our soft deletion support. We should have cleaned up all sites we may have put in a bad state by error. But if the error persists, please [reach out to support](https://support.github.com/contact) and we will be able to assist promptly.

**Reply by salvaRC** [reply]
Thanks for reaching out... It is solved now :)

**Reply by Hamzawp** [reply]
@salvaRC I am facing the same problem, can you help me with this?
I have a pro plan.
@yoannchaudet @queenofcorgis

**Comment by Bryson-Tai** [comment]
> I suspect this is related to an issue we had regarding our soft deletion support. We should have cleaned up all sites we may have put in a bad state by error. But if the error persists, please [reach out to support](https://support.github.com/contact) and we will be able to assist promptly.

Hi @yoannchaudet, my page is still unable to view properly even I am in public state. Kindly provide a guide on how to solve it. Thanks :D

**Reply by yoannchaudet** [reply]
Contact support: https://support.github.com/contact
